[safe-merge-queue] Update frequency of periodic scheduler

briannafan opened the pull request:
#### 5e03ba326279956ab44d2006ff0b6b917401b64c
<pre>
[safe-merge-queue] Update frequency of periodic scheduler
<a href="https://bugs.webkit.org/show_bug.cgi?id=262632">https://bugs.webkit.org/show_bug.cgi?id=262632</a>
rdar://116472019

Reviewed by Jonathan Bedard.

Updated the trigger period to 15 minutes.

* Tools/CISupport/ews-build/config.json:

Canonical link: <a href="https://commits.webkit.org/268939@main">https://commits.webkit.org/268939@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/0087445acdcc4ac915075baa598a1c2602a8bd46

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/21090 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/21467 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/22157 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/22971 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/19621 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/21330 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/24724 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/21656 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/20862 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/21314 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/21035 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/18292 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/23826 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/18193 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/19120 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/25392 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/19274 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/19322 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/23323 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/19867 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/16885 "Passed tests") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/20912 "Passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/19143 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/23437 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/2611 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/19718 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

JonWBedard commented on Tools/CISupport/ews-build/config.json:
```diff
@@ -433,7 +433,7 @@
     },
     {
-      "type": "Periodic", "name": "safe-merge-queue", "periodicBuildTimer": 7200,
+      "type": "Periodic", "name": "safe-merge-queue", "periodicBuildTimer": 1800,
```
I recommended running this bot every 30 minutes, @aj062, any thoughts?

aj062 replied:
30 or even less, maybe 10 or 15 minutes. Considering that in future this could be the recommended approach for engineers to use safe-merge-queue by default, If someone adds this label on a PR with everything green, it would be wasteful to wait 30 minutes (in worst case) for almost no reason.

JonWBedard replied:
@briannafan, if @aj062 want's to go more frequent, 15 minutes sounds reasonable to me.

briannafan committed:
[safe-merge-queue] Update frequency of periodic scheduler
https://bugs.webkit.org/show_bug.cgi?id=262632
rdar://116472019

Reviewed by Jonathan Bedard.

Updated the trigger period to 15 minutes.

* Tools/CISupport/ews-build/config.json:

Canonical link: https://commits.webkit.org/268939@main
